reactjs-todo/e2e/embedded-login.spec.js

```diff
@@ -0,0 +1,23 @@
+import { test, expect } from '@playwright/test';
+
+test('embedded login react', async ({ page }) => {
+  await page.goto('https://localhost:8443/', { waitUntil: 'networkidle' });
+  await page.getByRole('link', { name: 'Sign In', exact: true }).click();
+
+  //failed login
+  await page.getByLabel('User Name').fill('notauser');
+  await page.getByLabel('Password').fill('notapassword');
+
+  await page.getByPlaceholder('Password').press('Enter');
+  await expect(page.getByText('Login failure')).toBeVisible();
+
+  //successful login
+  await page.goto('https://localhost:8443/', { waitUntil: 'networkidle' });
```

Comment:
same as above

Reply:
set to default and worked(load)

Reply:
I set the 'waitUntil' option to 'load' which is the default option and worked as expected.